dataprofiler/profilers/profiler_options.py

```diff
@@ -145,6 +148,17 @@ def __init__(self):
         """
         super().__init__(is_enabled=True)
 
+    def _validate_helper(self, variable_path='BaseColumnOptions'):
+        """
+        Validates the options do not conflict and cause errors.
+
+        :param variable_path: current path to variable set.
+        :type variable_path: str
+        :return: list of errors (if raise_error is false)
+        :rtype: list(str)
+        """
+        return super()._validate_helper(variable_path) 
+
```

Comment:
It actually ends up being necessary because of the default argument (variable_path = "BaseColumnOptions") without it, all the variable paths end up being incorrect.

Reply:
good point!